app/assets/stylesheets/home.scss

```diff
@@ -38,15 +39,15 @@ h1, h2, h3{
     &.lostpet{
 
       h2{
-        margin-left: 149px;
-        margin-top: 15px;
+        margin-left: 200px;
+        // margin-top: 15px;
```

Comment:
Please delete this line. We shouldn't leave commented code when it can just be deleted. Don't forget that Git will keep of history of all the code you've changed